tests/acceptance/features/coreApiShareOperationsToShares2/shareAccessByID.feature

```diff
@@ -121,35 +107,3 @@ Feature: share access by ID
       | ocs_api_version | ocs_status_code | declined_share_path   |
       | 1               | 100             | /Shares/textfile0.txt |
       | 2               | 200             | /Shares/textfile0.txt |
-
-
-  Scenario Outline: decline a share using a invalid share Id
-    Given parameter "shareapi_auto_accept_share" of app "core" has been set to "yes"
-    And using OCS API version "<ocs_api_version>"
-    When user "Brian" declines share with ID "<share_id>" using the sharing API
-    Then the OCS status code should be "404"
-    And the HTTP status code should be "<http_status_code>"
-    And the API should not return any data
-    Examples:
-      | ocs_api_version | share_id   | http_status_code |
-      | 1               | 2333311    | 200              |
-      | 2               | 2333311    | 404              |
-      | 1               | helloshare | 200              |
-      | 2               | helloshare | 404              |
-      | 1               | $#@r3      | 200              |
-      | 2               | $#@r3      | 404              |
-      | 1               | 0          | 200              |
-      | 2               | 0          | 404              |
-
-
-  Scenario Outline: decline a share using empty share Id
-    Given parameter "shareapi_auto_accept_share" of app "core" has been set to "yes"
-    And using OCS API version "<ocs_api_version>"
-    When user "Brian" declines share with ID "" using the sharing API
-    Then the OCS status code should be "<ocs_status_code>"
-    And the HTTP status code should be "<http_status_code>"
-    And the API should not return any data
-    Examples:
-      | ocs_api_version | http_status_code | ocs_status_code |
-      | 1               | 200              | 999             |
-      | 2               | 500              | 500             |
```

Comment:
why remove this scenario? Doesn't it work fine in ocis without occ command

Reply:
these scenarios are in expected failure.

Reply:
These could be valid test scenarios. Just remove step `Given parameter "shareapi_auto_accept_share" of app "core" has been set to "yes"`. IMO, the tests are in expected failure list not because of that Given step.